### DISCUSSION TITLE
Download more than one image at a time

### DISCUSSION THREAD

**Discussion by bucklevision** [jomjol/AI-on-the-edge-device#2416] [General] [answered]
### The Feature

I cannot find instruction anywhere that tells me if it's possible to download the large number of images taken while recording digits for model training. I have a log/digits folder that is now full of folders and images, but I seem to have to download them one at a time?

**Comment by caco3** [comment, marked as answer]
This is currently not supported.

You could take out the SD-Card and copy the files from there.

Or you could open the file server and use a browser plugin to fetch all images in that folder:
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/1783586/a4e83971-94b8-49a7-ba0c-8e88c58168ae)


I use https://addons.mozilla.org/de/firefox/addon/downthemall/ :
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/1783586/fcee35f9-8b2a-47a9-ac81-ef2113f75ca3)

**Comment by bucklevision** [comment]
Yes I realise its not supported and there's workarounds, but that's why I created it as a feature request 😂
